src/panels/lovelace/types.ts

```diff
@@ -49,6 +49,7 @@ export interface LovelaceCard extends HTMLElement {
   hass?: HomeAssistant;
   isPanel?: boolean;
   preview?: boolean;
+  layout?: string;
```

Comment:
Should we type this more strict?

Reply:
For now it's only grid but custom section or view can provide other layout so we don't really know what it will be.

Reply:
But our cards will never adjust to layout settings provided by custom sections/views. So I would type it to the values we will deal with.